operator/cmd/mesh/upgrade.go

```diff
@@ -83,8 +81,6 @@ type upgradeArgs struct {
 func addUpgradeFlags(cmd *cobra.Command, args *upgradeArgs) {
 	cmd.PersistentFlags().StringSliceVarP(&args.inFilenames, "filename",
 		"f", nil, "Path to file containing IstioOperator custom resource")
-	cmd.PersistentFlags().StringVarP(&args.versionsURI, "versionsURI", "u",
```

Comment:
will removing flags here break user scripts? What is our policy here?

Reply:
In theory yes. In practice I cannot image anyone actually using this flag. Basically we have a yaml file with a bunch of version checks ("Only support upgrade from 1.x.y"). If you don't meet those requirements, you can also pass `--force` to skip it. This flag adds a way to reference a completely different yaml file from a url with a bunch of different version checks. Based on that I am inclined to remove it directly -- just silently doing nothing may be more confusing here